admin/src/main/resources/io/buoyant/admin/js/delegator.js

```diff
@@ -29,7 +29,7 @@ var Delegator = (function() {
       var path = $('#path-input').val();
       window.location.hash = encodeURIComponent(path);
       var request = $.get(
-        "/delegator.json?" + $.param({ n: path, d: dtabViewer.dtabStr() }),
+        "/delegator.json?" + $.param({ path: path, dtab: dtabViewer.dtabStr() }),
```

Comment:
pretty sure we're going to have to move this to a POST api in the near future...  When dtabs have comments, etc, this is going to be fragile.